shell/shell.c

```diff
@@ -897,6 +897,7 @@ static void set_view_type(ShellViewType viewtype, gboolean reload)
 
     shell->normalize_percentage = TRUE;
     shell->view_type = viewtype;
+    shell->_order_type = 0;
```

Comment:
The benchmarks each specifically set OrderType in the shell params. Filesystems does not, so it ends up using whatever the last one was. Resetting to 0 (descending) and having that as a default seems like the right fix, considering that 1 (ascending) has that special handling.